js/map.js

```diff
@@ -79,16 +84,37 @@ const createMarker = (ad) => {
     .bindPopup(createPopup(ad));
 };
 
-const getAds = getData(createMarker, COUNT_OF_ADS);
-getAds();
-
 const resetMainPin = () => {
   mainPinMarker.setLatLng(MAIN_LOCATION);
-  map.setView(MAIN_LOCATION, 13);
+  map.setView(MAIN_LOCATION, MAP_ZOOM);
   map.closePopup();
 };
 
-//loadSimilarPins(map);
-//};
+let allAds = getAds();
+
+(async function () {
+  allAds = await getAds();
+  allAds.slice(0, COUNT_OF_ADS).forEach((ad) => {
+    createMarker(ad);
+    toggleMapFiltersDisabled(false);
+  });
+})();
+
+
+const filterForm = document.querySelector('.map__filters');
+
+const filterAd = () => {
+  markerGroup.clearLayers();
+
+  const filteredAds = allAds.filter((ad) => checkAllFilters(ad));
+
+  filteredAds.slice(0, COUNT_OF_ADS).forEach((ad) => {
+    createMarker(ad);
+  });
+
+  if (filteredAds.length <= 0) {showAlert('Не удалось найти подходящие объявления');}
```

Comment:
```suggestion
  if (filteredAds.length <= 0) {
    showAlert('Не удалось найти подходящие объявления');
  }
```
делай сразу красиво)